pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/DSMConfig.java

```diff
@@ -23,6 +24,28 @@ public static String getSqlFromConfig(@NonNull String queryName) {
         return config.getString(queryName);
     }
 
+    public static Integer getIntFromConfig(@NonNull String name) {
+        try {
+            if (hasConfigPath(name)) {
+                return config.getInt(name);
+            }
+            return null;
+        } catch (Exception e) {
+            throw new DsmInternalError("Error getting int from config for " + name, e);
```

Comment:
I think the existing exceptions that typesafe throws when it can't find a key convey the same information to the caller.  Does throwing a wrapped DsmInternalError buy us some different exception handling here?

Reply:
Not really but this is how most of the other DSM exceptions are wrapped, and the server knows how to handle and communicate that to the frontend if needed.